### ISSUE TITLE
Improve selection behavior.

### ISSUE THREAD

**Issue by Georgios1999** [w-ahmad/WinUI.TableView#222]
**Is your feature request related to a problem? Please describe.**
Selecting a column can take more clicks than needed and cause confusion. Specifically:
- Keyboard navigation does not work with Enter to edit a field and only supports left/right arrow or tab to move between columns, effectively rendering it useless.
- The white focus border appears on the element when clicked, which can be visually unappealing. This also means the user will need to click again one or two times the already selected column.
- This gets worse with columns like the Combobox column, where it can take up to five clicks to edit it.

**Describe the solution you'd like**
Improve selection behavior so that:
- We have the ability to edit properties (color, transparency or thickness) of the focus border or simply remove it
- We have the ability to disable the extra focus required to navigate (single-click to select, double-click to edit)
- We have the option to make the Chevron on the Combobox column always visible and the column itself expand for edits in one click
- Improve keyboard navigation so that Enter edits a field and escape disengages focus

**Describe alternatives you've considered**
- Trying to set the focus border color to transparent (didn't work)

**Additional context**
###### I'm on mobile. May add screenshots when possible.
Closes discussion #157
